tests/leaderelection_test.go

```diff
@@ -0,0 +1,142 @@
+package tests_test
+
+import (
+	"regexp"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"kubevirt.io/containerized-data-importer/tests"
+	"kubevirt.io/containerized-data-importer/tests/framework"
+)
+
+const (
+	cdiDeploymentName = "cdi-deployment"
+
+	pollingInterval = 2 * time.Second
+	timeout         = 90 * time.Second
+)
+
+var (
+	logCheckLeaderRegEx = regexp.MustCompile("attempting to acquire leader lease  kube-system/cdi-controller-leader-election-helper...")
```

Comment:
one redundant white-space in "lease  kube-system" :)

Reply:
This string is from client-go and have to match it exactly.  But I think it may be better to match on strings we have control over.  Will change.